### DISCUSSION TITLE
Support for RSNA IAIP Transactions

### DISCUSSION THREAD

**Discussion by IntegratorBrad** [Project-MONAI/monai-deploy-app-sdk#332] [General]
Three profiles with four actors have been identified for RSNA IAIP testing:
* [AIR](https://www.ihe.net/uploadedFiles/Documents/Radiology/IHE_RAD_Suppl_AIR.pdf)
  * Evidence Creator
    * RAD-43 Store Evidence Documents
    * RAD-18 Creator Images Stored
    * RAD-29 Key Image Note Stored
    * RAD-108 Store Instances over the Web
* [AIW-I](https://www.ihe.net/uploadedFiles/Documents/Radiology/IHE_RAD_Suppl_AIW-I.pdf)
  * Task Manager
    * RAD-80 Create UPS Workitem
    * RAD-81 Query UPS Workitem
    * RAD-82 Claim UPS Workitem
    * RAD-83 Get UPS Workitem
    * RAD-84 Update UPS Workitem
    * RAD-85 Complete UPS Workitem
    * RAD-86 Manage UPS Subscription
    * RAD-87 Send UPS Notification
    * RAD-109 Open Event Channel
  * Task Performer
    * RAD-16 Retrieve Images
    * RAD-43 Evidence Documents Stored
    * RAD-80 Create UPS Workitem
    * RAD-81 Query UPS Workitem
    * RAD-82 Claim UPS Workitem
    * RAD-83 Get UPS Workitem
    * RAD-84 Update UPS Workitem
    * RAD-85 Complete UPS Workitem
    * RAD-87 Send UPS Notification
    * RAD-107 WADO-RS Retrieve
    * RAD-108 Store Instances over the Web
    * RAD-109 Open Event Channel
* [SOLE](https://www.ihe.net/uploadedFiles/Documents/Radiology/IHE_RAD_Suppl_SOLE.pdf) + [IAIP SOLE Extensions](https://docs.google.com/spreadsheets/d/1m-0XlQqnMSrGQL-CzsAxl5pL2W3w0uK81wjggzkQQ3Q/edit#gid=0 )
  * Event Reporter
    * ITI-20 Record Audit Event
    * RAD-124 Transfer Multiple Events

We will meet these requirements through two teams
* Team Fleischner - as an AI Orchestrator through MONAI Deploy (with Lung CT algorithm)
* Team Dotter - as an AI Performer through AIDE with MONAI Deploy (with 3 downstream algorithms)

**Comment by MMelQin** [comment]
I have a proposal for the "AI Orchestrator" role using custom app and workflow in MONAI Deploy, There are identified gaps, and we can discuss it in more details offline.
